openstack/objectstorage/v1/objects/requests.go

```diff
@@ -328,22 +329,41 @@ func (opts CopyOpts) ToObjectCopyQuery() (string, error) {
 
 // Copy is a function that copies one object to another.
 func Copy(c *gophercloud.ServiceClient, containerName, objectName string, opts CopyOptsBuilder) (r CopyResult) {
-	url, err := copyURL(c, containerName, objectName)
```

Comment:
moving this just a bit down below, just because `url` shadows the `net/url` package which we need for escaping.